src/java.desktop/share/classes/javax/swing/table/TableCellEditor.java

```diff
@@ -27,13 +27,11 @@
 
 import java.awt.Component;
 import javax.swing.CellEditor;
-import javax.swing.*;
+import javax.swing.JTable;
 
 /**
  * This interface defines the method any object that would like to be
- * an editor of values for components such as <code>JListBox</code>,
- * <code>JComboBox</code>, <code>JTree</code>, or <code>JTable</code>
- * needs to implement.
+ * an editor of values for the {@code JTable} component needs to implement.
```

Comment:
"This interface must be implemented to provide an editor of cell values for a {@code JTable}" ?

Reply:
Clear and to the point.

Reply:
I wonder if it makes sense to update the Javadoc for `TreeCellEditor` for consistency.

Reply:
Yeah, this description is clear and to the point. Will updated the PR.